### DISCUSSION TITLE
Turning on/off wallpaper zooming in Recent App List

### DISCUSSION THREAD

**Discussion by MehrabShawon** [MonwF/customiuizer#46] [Ideas]
Hello
I've customized my miui as my likings using customiuizer. The only problem I currently have with miui is wallpaper zooming. Whenever I go to recent app list, the wallpaper zooms a bit in the background. It's annoying for me. Recent system launcher versions have option for turning off/on wallpaper zooming, but recent launchers aren't much stable in my rom. So I use an older launcher (moded 4.39.0.5184) and it doesn't have wallpaper zooming off option. Can you add this feature in Customiuizer? It'll be really helpful!
Thanks for the time. Have a great day.

**Comment by MonwF** [comment]
have a look at https://github.com/qqlittleice/MiuiHome_R/tree/main/app/src/main/java/com/yuk/miuiHomeR/hook

**Reply by MehrabShawon** [reply]
Can't understand properly as I just started college recently and high school only teached me basic stuffs : (
Any easier way?
